src/api/vdom.js

```diff
@@ -1,22 +1,20 @@
-import * as IncrementalDOM from 'incremental-dom';
-import * as skateSymbols from './symbols';
-import { shadowDomV0, shadowDomV1 } from '../util/support';
-
-// Could import these, but we have to import all of IncrementalDOM anyways so
-// that we can export our configured IncrementalDOM.
-const {
+import {
   applyProp,
   attr,
   attributes,
   elementClose,
   elementOpen,
   elementOpenEnd,
   elementOpenStart,
+  elementVoid,
   skip,
   symbols,
-  text
-} = IncrementalDOM;
+  text,
+} from 'incremental-dom';
+import * as skateSymbols from './symbols';
+import { shadowDomV0, shadowDomV1 } from '../util/support';
 
+const fallbackToV0 = !shadowDomV1 && shadowDomV0;
```

Comment:
I thought about that but decided against it since this is the only pace it's used. Happy to refactor later.